### ISSUE TITLE
Feat: Select model per config

### ISSUE THREAD

**Issue by dragetd** [carlrobertoh/CodeGPT#104]
With GPT4 API access now slowly rolling out, it is really cool to have some more complex tasks as quick configs. The downside is the really slow generation of text.

Maybe we could attach a specific model to each of the configs?

**Comment by carlrobertoh** [comment]
Hey,

I'm not sure if I understand the feature request correctly. 
Are you suggesting attaching a specific model to each configured action prompt individually?

**Comment by dragetd** [comment]
Sorry, let me try to phrase it differently:

Currently I can use templates with '{{selectedCode}}' and open a chat with a menu command or hotkey.
It would be cool, if in the configuration I could not only set 'Action' and 'Prompt', but also 'Model'. So I could configure an action prompt to use gpt-4 (while my default model is gpt-3)

This is just a suggestion of course. No harm in closing the issue if it is nothing that can be done right now.

I haven't tried developing IDEA plugins yet, so I struggle with even ideas where to start adding such a thing myself.

**Comment by LaurentOngaro** [comment]
I agree, choosing the model in the configuration will be appreciated

**Comment by chlupac** [comment]
Or possibility of quick change current model
